cli/tests/error_003_typescript.ts

```diff
@@ -1,26 +1,20 @@
 /* eslint-disable */
-interface Value<T> {
-  f?: (r: T) => any;
-  v?: string;
-}
-
-interface C<T> {
-  values?: (r: T) => Array<Value<T>>;
-}
-
-class A<T> {
-  constructor(private e?: T, public s?: C<T>) {}
-}
-
-class B {
-  t = "foo";
-}
-
-var a = new A(new B(), {
-  values: o => [
-    {
-      v: o.t,
-      f: x => "bar"
+let x = {
+  a: {
+    b: {
+      c() {
+        return { d: "hello" };
+      }
     }
-  ]
-});
+  }
+};
+let y = {
+  a: {
+    b: {
+      c() {
+        return { d: 1234 };
+      }
+    }
+  }
+};
+x = y;
```

Comment:
🙄

Reply:
Our previous test was fixed in 3.6.3, so I had to replace it and @DanielRosenwasser suggested this one (though it is likely to get less complex in 3.7, so...). Just wanted something obnoxious to test.